tests/e2e/sample-applications/virt-todo/test.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+from datetime import datetime
+import json
+import requests
+
+
+#base_url = "http://localhost:8000"
+base_url = "http://vmroute-virttodo.apps.cluster-wdh03122024metala.wdh03122024metala.mg.dog8code.com"
+# example remote
+# base_url = "http://todolist-route-mysql-persistent.apps.cluster-wdh01102024g.wdh01102024g.mg.dog8code.com"
```

Comment:
maybe some note or link to doc on how this can be obtained

Reply:
it's in the README.MD